docs/config/README.md

```diff
@@ -384,11 +384,11 @@ The module is only visible when the device's battery is below 10%.
 
 | Option               | Default                           | Description                                         |
 | -------------------- | --------------------------------- | --------------------------------------------------- |
-| `full_symbol`        | `" "`                            | The symbol shown when the battery is full.          |
-| `charging_symbol`    | `" "`                            | The symbol shown when the battery is charging.      |
-| `discharging_symbol` | `" "`                            | The symbol shown when the battery is discharging.   |
-| `unknown_symbol`     | `" "`                            | The symbol shown when the battery state is unknown. |
-| `empty_symbol`       | `" "`                            | The symbol shown when the battery state is empty.   |
+| `full_symbol`        | `"• "`                            | The symbol shown when the battery is full.          |
+| `charging_symbol`    | `"⇡ "`                            | The symbol shown when the battery is charging.      |
+| `discharging_symbol` | `"⇣ "`                            | The symbol shown when the battery is discharging.   |
+| `unknown_symbol`     | `"❓ "`                            | The symbol shown when the battery state is unknown. |
+| `empty_symbol`       | `"❗ "`                            | The symbol shown when the battery state is empty.   |
```

Comment:
I think all of these should use some sort of battery emoji or a battery emoji combined with a status emoji, in the past the meaning of these symbols wasn't clear enough without a battery context.

Reply:
Might be. I just copied this over from the similar spaceship prompt.

Do you have any specific emojis in mind for this?

Reply:
🔋 and ⚡️ from the configuration example  could work. There's also a low battery emoji in Unicode 14.0, but that's still too new.